src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,304 @@
+package duke;
+
+import duke.fileHandling.FileHandler;
+import duke.task.*;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+
+public class Duke {
+    /**
+     * A divider (horizontal line).
+     */
+    public static final String DIVIDER_LINE_ONLY = "_____________________________________________________________";
+    /**
+     * A divider (horizontal line) with line break at the end.
+     */
+    public static final String DIVIDER = "_____________________________________________________________\n";
+
+    /**
+     * Prints greetings when the program starts.
+     */
+    public static void printGreetings() {
+        System.out.println(
+                DIVIDER
+                        + " Welcome.\n"
+                        + " I am your assistant Friday. ✧ʕ̢̣̣̣̣̩̩̩̩·͡˔·ོɁ̡̣̣̣̣̩̩̩̩✧\n"
+                        + " Just FYI, I am developed by Song Yu.\n"
+                        + " May I know what I can help you?\n"
+                        + DIVIDER_LINE_ONLY
+        );
+    }
+
+    /**
+     * Prints greetings when the program ends.
+     */
+    public static void printExitGreetings() {
+        System.out.print(
+                DIVIDER
+                        + " Thank you for getting in touch.\n"
+                        + " See you next time.\n"
+                        + "✧( ु•⌄• )◞ᴴᴬᵛᴱ ᴬ ᴳᴼᴼᴰ ᵀᴵᴹᴱ\n"
+                        + DIVIDER_LINE_ONLY
+        );
+    }
+
+    /**
+     * Prints the error message when user's input does not match any command.
+     */
+    public static void printNotCommand() {
+        System.out.println(
+                DIVIDER
+                        + " Sorry, your command is not recognized\n"
+                        + " See 'help'. \n"
+                        + DIVIDER_LINE_ONLY
+        );
+    }
+
+    /**
+     * Prints the help message.
+     */
+    public static void printHelp() {
```

Comment:
Perhaps, if possible, you could move printing- and task-related methods to another class to better organise your program, particularly your Duke class.